manifests/k/K1JT/wsjtx/v2.6.1/K1JT.wsjtx.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: K1JT.wsjtx
+PackageVersion: v2.6.1
+PackageLocale: en-US
+Publisher: Joe Taylor, K1JT
+PackageName: wsjtx

```

```suggestion
PackageName: WSJT-X: Digital Modes for Weak Signal Communications in Amateur Radio
```
This should match control panel